app/_src/gateway-operator/install.md

```diff
@@ -0,0 +1,14 @@
+---
+title: Install
+---
+
+{{ site.kgo_product_name }} can be installed using Kong provided Kubernetes manifests.
+
+If you intend to use {{ site.kic_product_name }} with {{ site.kgo_product_name }} you will need to install the KIC and Gateway API CRDs before installing {{ site.kgo_product_name }}:
+
+```bash
+kubectl apply -k https://github.com/Kong/kubernetes-ingress-controller/config/crd
+kubectl apply -k "https://github.com/kubernetes-sigs/gateway-api/config/crd?ref=v0.8.1"
```

Comment:
Has the version `v0.8.1` to be hardcoded? Could we use a similar approach like with the KIC version for gateway-api version too? If worth to do we can do it even as a separate PR

Reply:
I'm not sure if we can store more metadata in things like `{{ site }}` or `{{ page }}`.

There is some of that metadata stored in https://github.com/Kong/docs.konghq.com/blob/dac5b955bc6fe629caca9d6e255e42d22c43eb90/app/_plugins/generators/versions.rb#L34-L58 so that we do things like

```
{{ site.data.kong_latest_gateway.ee-version }}
```

where `.ee-version` is a field of `kong_latest_gateway`. I'm not sure how much effort that could be but feel free to contribute 🙃